civis/resources/_resources.py

```diff
@@ -404,9 +405,25 @@ def parse_path(path, operations, api_version, resources):
 
 
 def parse_swagger(swagger, api_version, resources):
-    """ Parse a swagger specifiction into a dictionary of classes
+    """ Dynamically create classes to interface with the Civis API.
+
+    Parse an OpenAPI (Swagger) specification into a dictionary of classes
     where each class represents an endpoint resource and contains
     methods to make http requests on that resource.
+
+    Parameters
+    ----------
+    api_spec : OrderedDict
```

Comment:
I started down the road of changing the language around swagger but I was worried there were too many changes to include it in the same PR as adding the cache functionality.  It occurs to me now that the best way to do this would be to first change that language in a separate PR and then rebase this on top.